toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -218,7 +218,6 @@ npth-1.6-4.cm2.aarch64.rpm
 pinentry-1.2.0-1.cm2.aarch64.rpm
 gnupg2-2.3.3-1.cm2.aarch64.rpm
 gpgme-1.16.0-1.cm2.aarch64.rpm
-mariner-repos-preview-2.0-3.cm2.noarch.rpm
```

Comment:
This might break derivative builds, which rely on the worker chroot being able to scan the correct Mariner repo for dependencies. In our tooling it's happening inside `graphpkgfetcher` (and probably inside `imagepkgfetcher`), where we run TDNF insdie the worker chroot environment to populate the local cache before we start a package/image build.

Reply:
Thanks for the note. This is a temporary change to deal with some local problems from the new TDNF having changed the return codes/CLI interface. Do you know why other repos aren't present in the chroot but the preview ones?